recipes-core/base-files/base-files_3.%.bbappend

```diff
@@ -1,8 +1,20 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend_menderimage := "${THISDIR}/${PN}:"
 
 #TODO: ${@bb.utils.contains("MACHINE_FEATURES", "sd", "packagegroup-mender-sd", "", d)}
 #./meta/recipes-core/packagegroups/packagegroup-base.bb
-do_install_append () {
+
+do_patch_device() {
+    # Noop if we're not building a mender-image.
+    true
```

Comment:
Maybe @bboozzoo can comment here, but do we need this? I think I haven't seen anything similar so far...